elastic/kibana
# [Logstash] Remove formatted message component in favor of i18n translate function

## Summary

The Logstash pipelines page is generating console warnings because JSX is injected to `EuiInMemoryTable`'s `search` prop where only a string is valid. The JSX was added to provide internationalization of the page's controls, which is working, but [has caused other problems](https://github.com/elastic/kibana/pull/43162) in addition to the dirty console.

## Testing this PR
0. Run Elasticsearch with security enabled, and start a trial
1. Navigate to `Management/Logstash Pipelines`
2. Check the console log, make sure that there are no errors/warnings related to the page

### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.


### For maintainers
